regression-test/suites/inverted_index_p1/tpcds_sf1_index/ddl/call_center.sql

```diff
@@ -1,8 +1,8 @@
 CREATE TABLE IF NOT EXISTS call_center (
     cc_call_center_sk bigint,
     cc_call_center_id char(16),
-    cc_rec_start_date date,
-    cc_rec_end_date date,
+    cc_rec_start_date datev2,
```

Comment:
do not need do this, we should submit a PR, to allow create a column with date type, but the underlying type is datev2

Reply:
> do not need do this, we should submit a PR, to allow create a column with date type, but the underlying type is datev2

just commit something to run the test p1